lib/node_modules/@stdlib/constants/float32/sqrt-two-pi/test/test.js

```diff
@@ -0,0 +1,41 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var sqrtf = require( '@stdlib/math/base/special/sqrtf' );
+var PI = require( '@stdlib/constants/float32/pi' );
+var FLOAT32_SQRT_TWO_PI = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a number', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof FLOAT32_SQRT_TWO_PI, 'number', 'main export is a number' );
+	t.end();
+});
+
+tape( 'the exported value is a single-precision floating-point number equal to Square root of the `π` times `2`', function test( t ) {
+	var expected = sqrtf( 2 * PI );
+	t.equal( FLOAT32_SQRT_TWO_PI, expected, '\'returns expected value\'' );
+	t.end();
```

Comment:
```suggestion
var FLOAT32_SQRT_TWO_PI = require( './../lib' );


// TESTS //

tape( 'main export is a number', function test( t ) {
	t.ok( true, __filename );
	t.strictEqual( typeof FLOAT32_SQRT_TWO_PI, 'number', 'main export is a number' );
	t.end();
});

tape( 'the exported value is a single-precision floating-point number equal to Square root of the `π` times `2`', function test( t ) {
	t.equal( FLOAT32_SQRT_TWO_PI, 2.5066282749176025, 'returns expected value' );
	t.end();
```

Beware of strict equality with computed values. While unlikely to differ in this case, that may not be generally true.